xml/installation.xml

```diff
@@ -29,143 +26,31 @@
-     <term>&hpca; compute node</term>
-     <listitem>
-      <para>
-       This role includes the following features:
-      </para>
-      <itemizedlist>
-       <listitem>
-        <para>
-         Based on the minimal setup configuration
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Uses XFS as the default root file system
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Includes &hpca;-enabled libraries
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Disables firewall and Kdump services
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Installs a client for the &slurm; workload manager
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Does not create a separate <literal>/home</literal> partition
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Mounts a large scratch partition to <filename>/var/tmp</filename>
-        </para>
-       </listitem>
-      </itemizedlist>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term>&hpca; development node</term>
-     <listitem>
-      <para>
-       This role includes the following features:
-      </para>
-      <itemizedlist>
-       <listitem>
-        <para>
-         Includes &hpca;-enabled libraries
-        </para>
-       </listitem>
-       <listitem>
-        <para>
-         Adds compilers and development toolchains
-        </para>
-       </listitem>
-      </itemizedlist>
-     </listitem>
-    </varlistentry>
-   </variablelist>
+   <itemizedlist>
+    <listitem>
+     <para>
+      <literal>Development Tools Module</literal>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <literal>HPC Module</literal>
+     </para>
+    </listitem>
+   </itemizedlist>
    <para>
-    The scratch partition <literal>/var/tmp/</literal> is only created if
-    there is sufficient space available on the installation medium (minimum
-    32&nbsp;GB).
-   </para>
-   <para>
-    The environment module <literal>Lmod</literal> is installed for all
-    roles. It is required at build time and runtime of the system. For more
-    information, see <xref linkend="sec-compute-lmod"/>.
-   </para>
-   <para>
-    All libraries specifically built for &hpca; are installed under
-    <literal>/usr/lib/hpc</literal>. They are not part of the standard search
-    path, so the <literal>Lmod</literal> environment module system is
-    required.
-   </para>
-   <para>
-    <literal>&munge;</literal> authentication is installed for all roles. &munge;
-    keys are generated and must be copied to all nodes in the
-    cluster. For more information, see <xref linkend="sec-remote-munge"/>.
-   </para>
-   <para>
-    The system roles are only available for new installations of the &hpcm; for &sles;.
+    Use either the <literal>Text Mode</literal> or <literal>Minimal</literal> system role.
```

Comment:
It is recommended you use ...
Why? Maybe one should say so as well.
_In HPC systems take on different roles and individual installations vary widely. Therefore, SUSE does not provide pre-defined roles. It is recommended you start with either the Text Mode or Minimal system role and add components according to your needs._